libjlm/include/jlm/tooling/CommandLine.hpp

```diff
@@ -529,4 +529,4 @@ class JhlsCommandLineParser final : public CommandLineParser {
 
 }
 
-#endif //JLM_TOOLING_COMMANDLINE_HPP
\ No newline at end of file
+#endif //JLM_TOOLING_COMMANDLINE_HPP
```

Comment:
This is noise. There is no other change in this file.

Reply:
fixed